src/Analysis/Ast/Impl/Types/PythonFunctionOverload.cs

```diff
@@ -167,7 +167,7 @@ public PythonFunctionOverload(IPythonClassMember cm, FunctionDefinition fd, Loca
             }
 
             if (typeArgs != null) {
-                var specificReturnValue = returnClassType.CreateSpecificType(new ArgumentSet(typeArgs));
+                var specificReturnValue = returnClassType.CreateSpecificType(new ArgumentSet(typeArgs, args.Expression, args.Eval));
```

Comment:
If you run MethodFromBaseClass, this fails here, seemingly because `args` is null.

Reply:
GetReturnDocumentation calls with `null` args explicitly, which triggers the if statement above. I think it's probably on purpose, so maybe just `args?` here is good enough.

Reply:
Thanks! Fixed.